src/components/Dropdown/Dropdown.tsx

```diff
@@ -91,6 +111,16 @@ export function Dropdown<V>(props: Props<V>) {
         ...state.attributes.popper,
         ...elem('list'),
     };
+    const getItemPropsByIndex = (index: number) => {
+        return {
+            ...getItemProps({
+                index,
+                item: items[index],
+            }),
+            isHighlighted: highlightedIndex === index,
```

Comment:
Is this a ListItem specific prop? What happens when it is used on other components, as per your AC?